app/src/main/java/com/github/polybooks/AddSaleActivity.kt

```diff
@@ -10,9 +10,9 @@ import androidx.appcompat.app.AppCompatActivity
 const val ISBN = "com.github.polybooks.ISBN"
 
 
-/* This activity is the starting point to add a new sale. It gives the option between scanning the book barcode or manually inputting the ISBN.
+/** This activity is the starting point to add a new sale. It gives the option between scanning the book barcode or manually inputting the ISBN.
```

Comment:
What's the difference between those two styles of commenting?

Reply:
This marks it as javadoc and not just comments, meaning if you hover over the class (from another file) this description will appear

Reply:
Ok, thanks for the explanation and for already fixing it!